### PR TITLE
fixed name overflow on sprites

### PR DESCRIPTION
**Pull request by azure-kyuri**
added name overflow on long sprite names (#94 )

### CODE REVIEW

File: src/components/SpritePane/SpriteCard.tsx [hunk 1]
```diff
@@ -13,21 +13,50 @@ type SpriteCardProps = {
...
-            onClick={onClick}
-            key={target?.sprite?.name}
-            width={120}
-            height={120}
-        >
-            <CostumeImage costume={target?.sprite?.costumes[0]} className="costumeCardImage" />
-        </ItemCard>
+    return (
+        <>
+            {target?.sprite?.name.length > 13 ?
```
Comment (ElliotRoe) [inline src/components/SpritePane/SpriteCard.tsx:35]: Would probably refactor this ternary into an if statement in the body of the component's function. Just so we don't have to repeat a bunch of JSX

File: src/components/SpritePane/SpriteName.tsx [hunk 2]
```diff
@@ -37,17 +37,21 @@ export function SpriteName() {
         setName(editingTarget.sprite.name);
     }, [editingTarget]);
 
-    return(<Grid display="flex">
-        <HorizontalButtons sx={{marginBottom: '12px'}}>
-            <TextField 
+    useEffect(() => {
+        console.log("user is typing")
+    }, [name])
```
Comment (ElliotRoe) [inline src/components/SpritePane/SpriteName.tsx:42]: Remove debug statements

**Review by ElliotRoe — CHANGES_REQUESTED** [review]
Looks good, just need to refactor the ternary and remove some code used for debugging

**Comment by azure-kyuri** [comment]
I refactored the ternary operator and also removed the debug statement in spriteName.tsx.

**Review by ElliotRoe — APPROVED** [review]

**Pull request merged**